readme.md

```diff
@@ -34,20 +34,20 @@ import 'package:grinder/grinder.dart';
 main(args) => grind(args);
 
 @Task('Initialize stuff.')
-init(GrinderContext context) {
-  context.log("Initializing stuff...");
+init() {
+  log("Initializing stuff...");
 }
 
 @Task('Compile stuff.')
 @Depends(init)
-compile(GrinderContext context) {
-  context.log("Compiling stuff...");
+compile() {
+  log("Compiling stuff...");
 }
 
 @DefaultTask('Deploy stuff.')
```

Comment:
The reason I went with `@DefaultTask` is to make it stick out as much as possible from the other tasks, since:

* It's the most frequently run (thus important) task.
* There can only be one, so it should be easy to scan to verify that.
* To avoid confusion that the task has the same name as the method, as with non-default tasks.

For example:

```dart
@Task('''
This is an awesome task.
You should run it all of the time.''', isDefault: true)
awesome() {
  // ...
}
```

would be easy to misread as just a regular `awesome` task.  That seems less likely with:

```dart
@DefaultTask('''
This is an awesome task.
You should run it all of the time.''')
awesome() {
  // ...
}
```